site/content/docs/user/ingress.md

```diff
@@ -165,8 +165,8 @@ kubectl apply -f {{< absURL "examples/ingress/usage.yaml" >}}
 Now verify that the ingress works
 
 {{< codeFromInline lang="bash" >}}
-# should output "foo"
-curl localhost/foo
-# should output "bar"
-curl localhost/bar
+# should output "foo-app"
+curl localhost/foo/hostname
+# should output "bar-app"
```

Comment:
we can iterate on this, but echo?msg=foo and echo?msg=bar seem more appropriate.

Reply:
Hm, what's the aim of the curl command? To receiving the identity of these two pods, don't we? Instead, _echo_ will respond with everything you provide to _echo_.

```shell
(kind-kind:default)
[~]$ curl http://localhost/foo/echo\?msg\=bar
bar%
(kind-kind:default)
[~]$ curl http://localhost/foo/hostname
foo-app
```

But the bottom line is that I don't care (it's not meant to be rude). If you want me to add the echo instead of the hostname, that's fine with me :)

Reply:
good point